src/planning.py

```diff
@@ -258,11 +276,51 @@ def _skeleton_generator(
         else:
             # Generate successors.
             metrics["num_nodes_expanded"] += 1
+            # If an abstract ldl policy is provided, generate policy-based
```

Comment:
is this a repeat of the `run_policy_guided_astar` stuff in utils.py? is that just bc this entire `skeleton_generator` is really a repeat of `run_astar`?

Reply:
Double yes! If we did this using run_astar it would break other stuff, so we just updated skeleton_generator instead.